client/home.html

```diff
@@ -23,6 +23,42 @@
             </label>
 
             <input id="stream-search-input" type="search"/>
+
+            <div id="checkboxes" hidden="hidden">
+              <div>
+                <p>Filters</p>
+              </div>
+              <div id="div-deepstream">
+                <input id="deepstream-checkbox" type="checkbox" checked="true">
+                <label for="deepstream-checkbox">DeepStreams</label>
+              </div>
+
+              <div id="livestreams">
+                <input id="livestreams-checkbox" type="checkbox" checked="true"/>
+                <label for="livestreams-checkbox">LiveStreams</label>
+              </div>
+
+              <div  id="div-youtube">
+                <input id="youtube-checkbox" type="checkbox" checked="true"/>
+                <label for="youtube-checkbox">YouTube</label>
+              </div>
+
+              <div id="div-twitch" >
+                <input id="twitch-checkbox" type="checkbox"/>
+                <label for="twitch-checkbox" >Twitch</label>
+              </div>
+
+              <div id="div-ustream" >
+                <input id="ustream-checkbox" type="checkbox" checked="true"/>
+                <label for="ustream-checkbox">uStream</label>
+              </div>
+
+              <div  id="div-bambuser">
```

Comment:
a little awkward to use id's for all these divs when they are never referred to anywhere. ids can be risky in certain cases (if anyone inadvertently make another id="div-bambuser", some weird behavior could result since)